cli/src/template/init/src/lib.rs

```diff
@@ -1,5 +1,7 @@
-// Here we're calling a macro exported by uniffi. This macro will
-// write some functions and bind them to the uniffi UDL file. These
+use circom_prover::witness::WitnessFn;
```

Comment:
maybe we should put this in `// CIRCOM_TEMPLATE`
so if `circom` feature is not activated, it won't be an error